infrastructure/code/issue_enhancer.go

```diff
@@ -80,6 +106,9 @@ func (b *IssueEnhancer) addIssueActions(ctx context.Context, issues []snyk.Issue
 		return
 	}
 
+	// Populate HTML template
+	b.enhanceIssuesDetails(issues)
+
```

Comment:
suggestion: let's rename `addIssueActions` func to something like `addActionsAndDetails` or `enrichIssue`, as it's now having an unexpected side effect.

Reply:
@bastiandoetsch I think I found a better place for this enhancement; I made `Scanner` responsible for enhancing the issues before adding the issues to the cache. WDYT?

Reply:
Might have a slight performance penalty (iterating over issues again), but should be fine :)